### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_BlazorWasmHome_UseOfColor:Only underline used to differentiate the link for "Brief Survey" from normal text.

### ISSUE THREAD

**Issue by v-akr** [dotnet/aspnetcore#42437]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.1;#Win11;#Benchmark;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir BlazorWasmr"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new blazorwasm --hosted -au Individual" and hit enter.
5. Run command "cd Server" and hit Enter.
6. Run the following command "dotnet run"
7. Copy the First URL link from Command prompt and copy it
8. Paste it in URL section of Browser and hit Search
9. Browser page will open with above template loaded
10. Navigate to Home Page , Start Accessibility Insights color tool and observe the behavior of "Brief Survey" link.

### Actual:
Only underline used to differentiate the link for "Brief Survey" from normal text.

### Expected:
Color should be used to differentiate the link for "Brief Survey" from normal text.

### User Impact:
color blind user will face difficulty to get in information.

### Attachment : 
[42437.zip](https://github.com/dotnet/aspnetcore/files/8999243/42437.zip)

**Comment by MackinnonBuck** [comment]
@v-akr Forgive me, but would you be able to clarify how this is an accessibility issue? I see this relates to WCAG 1.4.1 (Use of Color), which reads:

> Ensure color is not the sole means of communicating information. Do not use color in a way that requires the user to discriminate between hues as the only means to convey information, indicate an action, prompt a response, or distinguish a visual element.

And the "User Impact" section of this issue states:

> color blind user will face difficulty to get in information.

But in this case, color is not the sole means of communicating information, because we're not using color _at all_ to communicate information (just the underline, as you've pointed out in this issue). But giving the link a different color as suggested won't help colorblind users, right? Wouldn't having an underline actually be a solution to this issue? Please correct me if I'm misunderstanding something!

**Comment by SteveSandersonMS** [comment]
I'll close this since it seems to be reported in error. Color blind users will certainly *not* be disadvantaged by this aspect of the styling (if anything, the reverse could be true).

@v-akr If needed please reopen with more details.

**Comment by mkArtakMSFT** [comment]
Discussed this with the accessibility test team member and we've agreed to lower the severity to #A11ySev4.
Based on this closing as we won't be able to get to this due to resourcing constraints.
